tensorflow/core/kernels/segment_reduction_ops.cc

```diff
@@ -470,9 +498,18 @@ class UnsortedSegmentSumOp : public UnsortedSegmentBaseOp<Device, T, Index> {
   explicit UnsortedSegmentSumOp(OpKernelConstruction* context)
       : UnsortedSegmentBaseOp<Device, T, Index>(
             context,
-            sum_functor_) {}
+            &sum_functor_) {
+    bool drop_negatives = false;
+    OP_REQUIRES_OK(context, context->GetAttr("drop_negatives", &drop_negatives));
+    if (drop_negatives) {
+      this->reduction_functor_ = &this->sum_with_drop_negatives_functor_;
```

Comment:
Do we need `this->`? If not, please remove.

Reply:
Thanks @drpngx. Done.